src/security/notifications/teams.py

```
@@ -80,10 +80,10 @@ def _post_to_teams(
             )
 
     script_dir = os.path.dirname(os.path.abspath(__file__))
-    send_script = os.path.join(os.path.dirname(script_dir), "send_to_teams.py")
+    send_script = os.path.join(os.path.dirname(script_dir), "send_notifications.py")
 
     if not os.path.exists(send_script):
-        logging.warning(f"send_to_teams.py not found at {send_script} – skipping {label.lower()}")
+        logging.warning(f"send_notifications.py not found at {send_script} – skipping {label.lower()}")
```

Ruff (0.15.10)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)